src/gen/gemini.py

```diff
@@ -0,0 +1,40 @@
+import json
+import google.generativeai as genai
+
+async def call_gemini(prompt="", api_key=None, generation_config=None, safety_settings=None):
+    genai.configure(api_key=api_key)
+    
+    if generation_config is None:
+        generation_config = {
+          "temperature": 0.9,
+          "top_p": 1,
+          "top_k": 32,
+          "max_output_tokens": 8192,
+        }
+
+    if safety_settings is None:
+        safety_settings = [
+            {
+                "category": "HARM_CATEGORY_HARASSMENT",
+                "threshold": "BLOCK_NONE"
+            },
+            {
+                "category": "HARM_CATEGORY_HATE_SPEECH",
+                "threshold": "BLOCK_NONE"
+            },
+            {
+                "category": "HARM_CATEGORY_SEXUALLY_EXPLICIT",
+                "threshold": "BLOCK_NONE"
+            },
+            {
+                "category": "HARM_CATEGORY_DANGEROUS_CONTENT",
+                "threshold": "BLOCK_NONE"
+            },
+        ]
+
+    model = genai.GenerativeModel(model_name="gemini-1.0-pro",
+                                generation_config=generation_config,
+                                safety_settings=safety_settings)
```

Comment:
Same. Shouldn't this be initialized just once?

Reply:
actually, no.

even if I didn't make CLI options for generation_config, sefety_settings yet, this should be configurable by users.

Reply:
Then "gemini-1.0-pro" should be a CLI option too.

Reply:
right. I don't want this too verbose for right now.
I am thinking about having another config.yaml to setup Gemini (model name, gen config, safe settings, etc.,). But want to handle this in the future PR. What do you think?

Reply:
Should be a single change:

`model_name="gemini-1.0-pro"` to `model_name=args.model_name`.

Reply:
yeah but don't want to put everything into CLI. There are too many parameters for gen_config, safety settings. If I have model_name, I feel like I need to add CLI options for the other two. 

not sure if this project is one time thing, but I am planning to add more service LLM like Claude and ChatGPT. so wanted to manage all relevant settings per service in separate files.

maybe just adding `model_name` CLI option for now, then work on generalization later?